docs/plugins/deploy.md

```diff
@@ -0,0 +1,35 @@
+# Deploy
+
+```
+serverless deploy --stage dev --region us-east-1
+```
+
+Deploys your service to AWS.
```

Comment:
you're right...actually our framework right now is still provider specific. For example when you deploy you don't get to choose a provider yet...but that can easily be changed I think